src/test/java/com/arpnetworking/commons/performance/BaseAccumulatorTestPerf.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2020 Dropbox
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.arpnetworking.commons.performance;
+
+import com.arpnetworking.commons.math.Accumulator;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Random;
+
+/**
+ * Base performance tests for the {@link Accumulator} implementations.
+ *
+ * @author Ville Koskela (ville at inscopemetrics dot io)
+ */
+public abstract class BaseAccumulatorTestPerf {
+
+    private static final List<Double> RANDOM_VALUE_DATA_SET;
+    private static final Logger LOGGER = LoggerFactory.getLogger(BaseAccumulatorTestPerf.class);
+
+    static {
+        // Random values data set
+        final List<Double> valueDataSet = new ArrayList<>();
+        final Random valueRandom = new Random(564564359);
```

Comment:
Is there a way to fix the seed across all benchmark cases but still let it vary between runs?

It would be useful IMO, at least for the benchmarking case, to be able to use a new seed for each run or optionally specify it via the command line arguments. I can understand the intent though, keeping the seed fixed gives a level playing field when comparing the algorithms.

Reply:
So yes, it's possible to extend these tests to provide more variance. The purpose for me was more for reproducible data set across tests as I fiddled with the implementations (e.g. streams are slow!). The variation in the data set comes from the values generated into the set; even pseudo-random I would expect it to be sufficient to exercise the algorithm. I think more than varying the seed, we should generated different data sets; e.g. bimodal data, non-uniform data, very small distributions, etc.. I'm going to punt on this for now.